build.sbt

```diff
@@ -10,37 +10,31 @@ lazy val runtime = (project in file("runtime"))
     crossScalaVersions := Seq("2.12.4", "2.11.11"),
     name := "GrpcGatewayRuntime",
     libraryDependencies ++= Seq(
-      "com.trueaccord.scalapb" %% "compilerplugin"          % "0.6.7",
-      "com.trueaccord.scalapb" %% "scalapb-runtime-grpc"    % "0.6.7",
       "com.trueaccord.scalapb" %% "scalapb-runtime-grpc"    % "0.6.7",
       "com.trueaccord.scalapb" %% "scalapb-json4s"          % "0.3.3",
       "io.grpc"                %  "grpc-netty"              % "1.8.0",
       "org.webjars"            %  "swagger-ui"              % "3.5.0",
-      "com.google.api.grpc"    % "googleapis-common-protos" % "0.0.3" % "protobuf"
-    ),
-    PB.protoSources in Compile += target.value / "protobuf_external",
```

Comment:
We don't need these settings, do we?

Reply:
This is where the `http` option is defined

Reply:
Yes, but everything works without that dependency. runtime doesn't need protos, they are generated at compile time